diff --git tycho-p2/tycho-p2-plugin/src/main/java/org/eclipse/tycho/plugins/p2/P2MetadataMojo.java tycho-p2/tycho-p2-plugin/src/main/java/org/eclipse/tycho/plugins/p2/P2MetadataMojo.java
index 63df0d3..6f02433 100644
--- tycho-p2/tycho-p2-plugin/src/main/java/org/eclipse/tycho/plugins/p2/P2MetadataMojo.java
+++ tycho-p2/tycho-p2-plugin/src/main/java/org/eclipse/tycho/plugins/p2/P2MetadataMojo.java
@@ -132,8 +132,7 @@ public class P2MetadataMojo extends AbstractMojo {
 
             for (Artifact attachedArtifact : project.getAttachedArtifacts()) {
                 if (attachedArtifact.getFile() != null
-                        && (attachedArtifact.getFile().getName().endsWith(".jar") || attachedArtifact.getFile()
-                                .getName().endsWith(".zip"))) {
+                        && (attachedArtifact.getFile().getName().endsWith(".jar")) {
                     artifacts.add(new ArtifactFacade(attachedArtifact));
                 }
             }
